### DISCUSSION TITLE
vscode插件形式的使用

### DISCUSSION THREAD

**Discussion by Eastboat** [Zhengqbbb/cz-git#176] [Q&A] [answered]
请问能不能像 cz-customizable 适配器 和Visual Studio Code Commitizen Support 插件这种结合使用
<img width="608" alt="image" src="https://github.com/Zhengqbbb/cz-git/assets/38577428/8b0e6d5c-ba06-494d-87ed-59a183e758ab">

**Comment by Zhengqbbb** [comment, marked as answer]
有想过，当时是想提交配置加载逻辑 PR 合并通用使用，但是这个仓库已经不再维护，而 VSCode 插件目前没有计划开发
https://github.com/KnisterPeter/vscode-commitizen

> 如果开发完成，会回来进行告知 📢📢📢

**Reply by Zhengqbbb** [reply]
当初把配置加载单独抽离成模块，就是为了提交给其他插件集成，但是万万没想到就 archive repo 了 🫠
https://www.npmjs.com/package/@cz-git/loader
https://github.com/Zhengqbbb/cz-git/tree/main/packages/%40cz-git/plugin-loader
